docs/reference/buildx_build.md

```diff
@@ -271,6 +271,148 @@ $ docker buildx build --cache-from=type=s3,region=eu-west-1,bucket=mybucket .
 
 More info about cache exporters and available attributes: https://github.com/moby/buildkit#export-cache
 
+### <a name="call"></a> Invoke a subrequest (--call)
+
+```text
+--call=[build|check|outline|targets]
+```
+
+BuildKit frontends can support multiple request types, also known as
+subrequests, in addition to the default build request. Subrequests provide
+alternative modes of execution for a build. These modes may or may not produce
+an artifact, depending on the purpose of the subrequest. They're useful for
+inspecting, validating, or generating alternative outputs from a build.
+
+The `--call` flag for `docker buildx build` lets you specify the type of
+request that you want to execute. If this flag is unspecified, it defaults to
+`build`, which executes the build and evaluates [build checks](https://docs.docker.com/reference/build-checks/).
+
+For Dockerfiles, the available request types are:
+
+| Command                        | Description                                                                       |
+| ------------------------------ | --------------------------------------------------------------------------------- |
+| `build` (default)              | Execute the build and run build checks.                                           |
```

Comment:
Technically this does not contain the `check` method. Warnings are showed only for your current build, while the `check` method by default will check warnings for your full Dockerfile.